tests/include/test/threading_helpers.h

```diff
@@ -58,17 +53,30 @@ void mbedtls_test_thread_set_alt(int (*thread_create)(mbedtls_test_thread_t *thr
                                                       void *thread_data),
                                  int (*thread_join)(mbedtls_test_thread_t *thread));
 
+#else /* MBEDTLS_THREADING_ALT*/
+
+typedef struct mbedtls_test_thread_t {
+
+#if defined(MBEDTLS_THREADING_PTHREAD)
+    pthread_t MBEDTLS_PRIVATE(thread);
+#else /* MBEDTLS_THREADING_PTHREAD */
+    /* Make sure this struct is always non-empty */
+    unsigned dummy;
+#endif
+
+} mbedtls_test_thread_t;
+
 #endif /* MBEDTLS_THREADING_ALT*/
 
 /**
- * \brief                   The function pointers for thread create and thread
+ * \brief The function pointers for thread create and thread
  *                          join.
```

Comment:
Minor, style, here and below: why are we only changing the alignment of the first line and not the others?
```suggestion
 * \brief The function pointers for thread create and thread join.
```